include/stl2/detail/functional/arithmetic/minus.hpp

```diff
@@ -0,0 +1,63 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter
+//  Copyright Christopher Di Bella
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#ifndef STL2_DETAIL_FUNCTIONAL_ARITHMETIC_MINUS_HPP
+#define STL2_DETAIL_FUNCTIONAL_ARITHMETIC_MINUS_HPP
+
+#include <stl2/detail/concepts/core.hpp>
+#include <stl2/detail/functional/arithmetic/plus.hpp>
+#include <stl2/detail/fwd.hpp>
+#include <stl2/detail/numeric_traits/identity.hpp>
+#include <type_traits>
+#include <utility>
+
+STL2_OPEN_NAMESPACE {
+	namespace ext {
+		template<class T, class U>
+		META_CONCEPT __differenceable_with = __summable_with<T, U> &&
+			requires(__uncvref<T> t, __uncvref<U> u) {
```

Comment:
I don't like that `T` and `U` have had their cv- and ref-qualifiers removed, but I can't think of a better way to implement this at the moment.